browser/src/models/GraphManager.tsx

```diff
@@ -40,14 +42,20 @@ export default class GraphManager {
       // classInfo is optional
     }
     try {
-      let manifestResponse = await axios.get(manifestUrl)      
-      this.componentSet = manifestResponse.data as ComponentSet;
+      try {
+        let manifestResponse = await axios.get(manifestUrl)
+        this.componentSet = manifestResponse.data as ComponentSet;
+      } catch {
+        let manifestResponse = await axios.get(gzippedManifestUrl, {responseType: 'arraybuffer', 'decompress': true })
```

Comment:
why not use gzip url immediately if it's set?  Consider throwing an error if both are set.  (or per above, just rely on suffix to know when to apply decompression)

Reply:
Problem is, we would need a way to pipe the extension from a config or build file. The way we currently do it is that we use the gzip url as a fallback if the json fetch fails. I'm not sure how to set it up so that we pass the extension at build time.